pulsar-broker/src/main/java/org/apache/pulsar/broker/service/Topic.java

```diff
@@ -129,6 +129,7 @@ default boolean isMarkerMessage() {
      */
     void recordAddLatency(long latency, TimeUnit unit);
 
+    @Deprecated
     CompletableFuture<Consumer> subscribe(TransportCnx cnx, String subscriptionName, long consumerId, SubType subType,
```

Comment:
There are a lot of tests called this interface, and in order to delete it in the future, should we change the test to call the new one?

Reply:
Okay, I will open another PR to do this